tests/mock_wp_functions.php

```diff
@@ -164,6 +166,69 @@ function wp_enqueue_style($handle, $src, $deps, $version, $media)
     MockWpEnqueue::add($handle, $src, $deps, $version, $media);
 }
 
+function get_term_by($field, $value, $taxonomy = '', $output = 'OBJECT', $filter = 'raw')
```

Comment:
is het logisch dat het argument $output hier een string is en in `get_term` een constant?

Reply:
Ik weet niet meer waarom dit het geval is. Ik heb ze voor nu allebei strings gemaakt want als je ze allebei constantes maakt gaat het stuk.